mappings/services.json

```diff
@@ -661,6 +661,11 @@
           "type": "string",
           "index": "not_analyzed"
         },
+        "service_id_hash": {
```

Comment:
Does this need to be added to the generator in the frameworks repo?

Reply:
https://github.com/alphagov/digitalmarketplace-frameworks/pull/475